modal_proto/api.proto

```diff
@@ -788,6 +788,7 @@ message SharedVolumeListFilesResponse {
 message SharedVolumeMount {
   string mount_path = 1;
   string shared_volume_id = 2;
+  CloudProvider cloud_provider = 3;
```

Comment:
shouldn't we add region to this too everywhere? might as well add that now

Reply:
yeah we'd need to do that. Probably worth waiting until we know how the regions will be setup and stuff though.